### ISSUE TITLE
Re-enable SSL checking

### ISSUE THREAD

**Issue by jeffbyrnes** [darkskyapp/madis#2]
This is bad juju: eb9723dbcb3393ea44d318e997613956eeceec4f

**Comment by jeffbyrnes** [comment]
@thegrossman @ironwallaby circling back around to this; seems like we should see if we can re-enable SSL checking. It’s only been, y’know, like a year.

That, or just close this out.

**Comment by ironwallaby** [comment]
I agree in theory but defer to @thegrossman—it's entirely possible that the MADIS server is _just broken_.
